Author: 
Description: This patch:
* removes all SGML declarations from the catalogs, for more info, see
debian/README.docbook.dcl
* removes the ISO entity pointers since they are supplied by sgml-data
* removes Public-IDs for DocBook XML
* forces Public-ID rather than the System-ID in DTD declarations
(closes: <URL:http://bugs.debian.org/97330>)

<URL:http://packages.debian.org/changelogs/pool/main/d/docbook/current/changelog#versionversion4.4-1>
<URL:http://packages.debian.org/changelogs/pool/main/d/docbook/current/changelog#versionversion4.1-5>
<URL:http://packages.debian.org/changelogs/pool/main/d/docbook/current/changelog#versionversion4.1-2>
<URL:http://packages.debian.org/changelogs/pool/main/d/docbook/current/changelog#versionversion3.0-4>
--- a/docbook-2.4.1/docbook.cat
+++ b/docbook-2.4.1/docbook.cat
@@ -27,8 +27,13 @@
   -- ...................................................................... --
   -- SGML declaration associated with DocBook ............................. --
 
-SGMLDECL "docbook.dcl"
-DTDDECL "-//Davenport//DTD DocBook V2.4.1//EN" "docbook.dcl"
+-- disabled, see README.docbook.dcl --
+-- SGMLDECL "docbook.dcl" --
+-- DTDDECL "-//Davenport//DTD DocBook V2.4.1//EN" "docbook.dcl" --
+
+  -- ...................................................................... --
+  -- Prefer PUBID to SYSID ................................................ --
+OVERRIDE YES
 
   -- ...................................................................... --
   -- DocBook driver file .................................................. --
@@ -43,28 +48,5 @@
 PUBLIC "-//Davenport//ELEMENTS DocBook Document Hierarchy V2.4.1//EN" "dbhier.mod"
 PUBLIC "-//Davenport//ELEMENTS DocBook Additional General Entities//EN" "dbgenent.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN" "iso-dia.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN" "iso-num.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN" "iso-pub.gml"
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN" "iso-tech.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN" "iso-lat1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN" "iso-lat2.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN" "iso-grk1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN" "iso-grk2.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN" "iso-grk3.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN" "iso-grk4.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN" "iso-amsa.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN" "iso-amsb.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN" "iso-amsc.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN" "iso-amsn.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN" "iso-amso.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN" "iso-amsr.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN" "iso-box.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN" "iso-cyr1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN" "iso-cyr2.gml"
-
   -- End of DocBook catalog data V2.4.1 ................................... --
   -- ...................................................................... --
--- a/docbook-3.0/docbook.cat
+++ b/docbook-3.0/docbook.cat
@@ -27,7 +27,12 @@
   -- ...................................................................... --
   -- SGML declaration associated with DocBook ............................. --
 
-DTDDECL "-//Davenport//DTD DocBook V3.0//EN" "docbook.dcl"
+-- disabled, see README.docbook.dcl --
+-- DTDDECL "-//Davenport//DTD DocBook V3.0//EN" "docbook.dcl" --
+
+  -- ...................................................................... --
+  -- Prefer PUBID to SYSID ................................................ --
+OVERRIDE YES
 
   -- ...................................................................... --
   -- DocBook driver file .................................................. --
@@ -42,28 +47,5 @@
 PUBLIC "-//Davenport//ELEMENTS DocBook Document Hierarchy V3.0//EN" "dbhier.mod"
 PUBLIC "-//Davenport//ENTITIES DocBook Additional General Entities V3.0//EN" "dbgenent.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN" "iso-dia.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN" "iso-num.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN" "iso-pub.gml"
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN" "iso-tech.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN" "iso-lat1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN" "iso-lat2.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN" "iso-grk1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN" "iso-grk2.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN" "iso-grk3.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN" "iso-grk4.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN" "iso-amsa.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN" "iso-amsb.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN" "iso-amsc.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN" "iso-amsn.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN" "iso-amso.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN" "iso-amsr.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN" "iso-box.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN" "iso-cyr1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN" "iso-cyr2.gml"
-
   -- End of catalog data for DocBook V3.0 ................................. --
   -- ...................................................................... --
--- a/docbook-3.1/docbook.cat
+++ b/docbook-3.1/docbook.cat
@@ -19,7 +19,12 @@
   -- ...................................................................... --
   -- SGML declaration associated with DocBook ............................. --
 
-DTDDECL "-//OASIS//DTD DocBook V3.1//EN" "docbook.dcl"
+-- disabled, see README.docbook.dcl --
+-- DTDDECL "-//OASIS//DTD DocBook V3.1//EN" "docbook.dcl" --
+
+  -- ...................................................................... --
+  -- Prefer PUBID to SYSID ................................................ --
+OVERRIDE YES
 
   -- ...................................................................... --
   -- DocBook driver file .................................................. --
@@ -36,28 +41,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Notations V3.1//EN" "dbnotn.mod"
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V3.1//EN" "dbcent.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN" "iso-dia.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN" "iso-num.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN" "iso-pub.gml"
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN" "iso-tech.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN" "iso-lat1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN" "iso-lat2.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN" "iso-grk1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN" "iso-grk2.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN" "iso-grk3.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN" "iso-grk4.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN" "iso-amsa.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN" "iso-amsb.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN" "iso-amsc.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN" "iso-amsn.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN" "iso-amso.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN" "iso-amsr.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN" "iso-box.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN" "iso-cyr1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN" "iso-cyr2.gml"
-
   -- End of catalog data for DocBook V3.1 ................................. --
   -- ...................................................................... --
--- a/docbook-4.0/docbook.cat
+++ b/docbook-4.0/docbook.cat
@@ -19,7 +19,12 @@
   -- ...................................................................... --
   -- SGML declaration associated with DocBook ............................. --
 
-DTDDECL "-//OASIS//DTD DocBook V4.0//EN" "docbook.dcl"
+-- disabled, see README.docbook.dcl --
+-- DTDDECL "-//OASIS//DTD DocBook V4.0//EN" "docbook.dcl" --
+
+  -- ...................................................................... --
+  -- Prefer PUBID to SYSID ................................................ --
+OVERRIDE YES
 
   -- ...................................................................... --
   -- DocBook driver file .................................................. --
@@ -36,28 +41,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Notations V4.0//EN" "dbnotn.mod"
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.0//EN" "dbcent.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN" "iso-dia.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN" "iso-num.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN" "iso-pub.gml"
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN" "iso-tech.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN" "iso-lat1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN" "iso-lat2.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN" "iso-grk1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN" "iso-grk2.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN" "iso-grk3.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN" "iso-grk4.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN" "iso-amsa.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN" "iso-amsb.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN" "iso-amsc.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN" "iso-amsn.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN" "iso-amso.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN" "iso-amsr.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN" "iso-box.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN" "iso-cyr1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN" "iso-cyr2.gml"
-
   -- End of catalog data for DocBook V4.0 ................................. --
   -- ...................................................................... --
--- a/docbook-4.1/docbook.cat
+++ b/docbook-4.1/docbook.cat
@@ -19,7 +19,12 @@
   -- ...................................................................... --
   -- SGML declaration associated with DocBook ............................. --
 
-DTDDECL "-//OASIS//DTD DocBook V4.1//EN" "docbook.dcl"
+-- disabled, see README.docbook.dcl --
+-- DTDDECL "-//OASIS//DTD DocBook V4.1//EN" "docbook.dcl" --
+
+  -- ...................................................................... --
+  -- Prefer PUBID to SYSID ................................................ --
+OVERRIDE YES
 
   -- ...................................................................... --
   -- DocBook driver file .................................................. --
@@ -36,28 +41,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Notations V4.1//EN" "dbnotn.mod"
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.1//EN" "dbcent.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN" "iso-dia.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN" "iso-num.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN" "iso-pub.gml"
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN" "iso-tech.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN" "iso-lat1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN" "iso-lat2.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN" "iso-grk1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN" "iso-grk2.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN" "iso-grk3.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN" "iso-grk4.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN" "iso-amsa.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN" "iso-amsb.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN" "iso-amsc.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN" "iso-amsn.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN" "iso-amso.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN" "iso-amsr.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN" "iso-box.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN" "iso-cyr1.gml"
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN" "iso-cyr2.gml"
-
   -- End of catalog data for DocBook V4.1 ................................. --
   -- ...................................................................... --
--- a/docbook-4.2/docbook.cat
+++ b/docbook-4.2/docbook.cat
@@ -22,9 +22,6 @@
 PUBLIC "-//OASIS//DTD DocBook V4.2//EN"
        "docbook.dtd"
 
-PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
-       "docbookx.dtd"
-
   -- ...................................................................... --
   -- DocBook modules ...................................................... --
 
@@ -49,65 +46,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.2//EN"
        "dbcentx.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN"
-       "iso-dia.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN"
-       "iso-num.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN"
-       "iso-pub.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN"
-       "iso-tech.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN"
-       "iso-lat1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN"
-       "iso-lat2.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN"
-       "iso-grk1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN"
-       "iso-grk2.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN"
-       "iso-grk3.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN"
-       "iso-grk4.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN"
-       "iso-amsa.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN"
-       "iso-amsb.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN"
-       "iso-amsc.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN"
-       "iso-amsn.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN"
-       "iso-amso.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN"
-       "iso-amsr.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN"
-       "iso-box.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN"
-       "iso-cyr1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN"
-       "iso-cyr2.gml"
-
   -- End of catalog data for DocBook V4.2 ................................. --
   -- ...................................................................... --
--- a/docbook-4.3/docbook.cat
+++ b/docbook-4.3/docbook.cat
@@ -22,9 +22,6 @@
 PUBLIC "-//OASIS//DTD DocBook V4.3//EN"
        "docbook.dtd"
 
-PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
-       "docbookx.dtd"
-
   -- ...................................................................... --
   -- DocBook modules ...................................................... --
 
@@ -49,65 +46,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.3//EN"
        "dbcentx.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN"
-       "iso-dia.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN"
-       "iso-num.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN"
-       "iso-pub.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN"
-       "iso-tech.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN"
-       "iso-lat1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN"
-       "iso-lat2.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN"
-       "iso-grk1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN"
-       "iso-grk2.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN"
-       "iso-grk3.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN"
-       "iso-grk4.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN"
-       "iso-amsa.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN"
-       "iso-amsb.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN"
-       "iso-amsc.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN"
-       "iso-amsn.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN"
-       "iso-amso.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN"
-       "iso-amsr.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN"
-       "iso-box.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN"
-       "iso-cyr1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN"
-       "iso-cyr2.gml"
-
   -- End of catalog data for DocBook V4.3 ................................. --
   -- ...................................................................... --
--- a/docbook-4.4/docbook.cat
+++ b/docbook-4.4/docbook.cat
@@ -22,9 +22,6 @@
 PUBLIC "-//OASIS//DTD DocBook V4.4//EN"
        "docbook.dtd"
 
-PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
-       "docbookx.dtd"
-
   -- ...................................................................... --
   -- DocBook modules ...................................................... --
 
@@ -52,65 +49,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.4//EN"
        "dbcentx.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN"
-       "iso-dia.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN"
-       "iso-num.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN"
-       "iso-pub.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN"
-       "iso-tech.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN"
-       "iso-lat1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN"
-       "iso-lat2.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN"
-       "iso-grk1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN"
-       "iso-grk2.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN"
-       "iso-grk3.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN"
-       "iso-grk4.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN"
-       "iso-amsa.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN"
-       "iso-amsb.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN"
-       "iso-amsc.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN"
-       "iso-amsn.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN"
-       "iso-amso.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN"
-       "iso-amsr.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN"
-       "iso-box.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN"
-       "iso-cyr1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN"
-       "iso-cyr2.gml"
-
   -- End of catalog data for DocBook V4.4 ................................. --
   -- ...................................................................... --
--- a/docbook-4.5/docbook.cat
+++ b/docbook-4.5/docbook.cat
@@ -22,9 +22,6 @@
 PUBLIC "-//OASIS//DTD DocBook V4.5//EN"
        "docbook.dtd"
 
-PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
-       "docbookx.dtd"
-
   -- ...................................................................... --
   -- DocBook modules ...................................................... --
 
@@ -52,65 +49,5 @@
 PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.5//EN"
        "dbcentx.mod"
 
-  -- ...................................................................... --
-  -- ISO entity sets ...................................................... --
-
-PUBLIC "ISO 8879:1986//ENTITIES Diacritical Marks//EN"
-       "iso-dia.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Numeric and Special Graphic//EN"
-       "iso-num.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Publishing//EN"
-       "iso-pub.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES General Technical//EN"
-       "iso-tech.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 1//EN"
-       "iso-lat1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Latin 2//EN"
-       "iso-lat2.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Letters//EN"
-       "iso-grk1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Monotoniko Greek//EN"
-       "iso-grk2.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Greek Symbols//EN"
-       "iso-grk3.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Alternative Greek Symbols//EN"
-       "iso-grk4.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Arrow Relations//EN"
-       "iso-amsa.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Binary Operators//EN"
-       "iso-amsb.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Delimiters//EN"
-       "iso-amsc.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Negated Relations//EN"
-       "iso-amsn.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Ordinary//EN"
-       "iso-amso.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Added Math Symbols: Relations//EN"
-       "iso-amsr.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Box and Line Drawing//EN"
-       "iso-box.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Russian Cyrillic//EN"
-       "iso-cyr1.gml"
-
-PUBLIC "ISO 8879:1986//ENTITIES Non-Russian Cyrillic//EN"
-       "iso-cyr2.gml"
-
   -- End of catalog data for DocBook V4.5 ................................. --
   -- ...................................................................... --
